pkg/operator/ceph/cluster/controller_test.go

```diff
@@ -112,6 +112,7 @@ func TestReconcile_DeleteCephCluster(t *testing.T) {
 		// Make sure it has the fake CephCluster that is to be deleted in it
 		client := clientfake.NewClientBuilder().WithScheme(scheme).WithRuntimeObjects(fakeCluster).Build()
 
+		assert.NoError(t, corev1.AddToScheme(scheme))
```

Comment:
for readability, better to separate the action from the test assert
```suggestion
		err := corev1.AddToScheme(scheme)
		assert.NoError(t, err)
```